olm-catalog/serverless-operator/1.9.0/serverless-operator.v1.9.0.clusterserviceversion.yaml

```diff
@@ -358,7 +358,7 @@ spec:
                     - name: IMAGE_DISPATCHER_IMAGE
                       value: registry.svc.ci.openshift.org/openshift/knative-v0.15.2:knative-eventing-channel-dispatcher
                     - name: IMAGE_KN_CLI_ARTIFACTS
-                      value: registry.svc.ci.openshift.org/openshift/knative-v0.14.0:kn-cli-artifacts
+                      value: registry.svc.ci.openshift.org/openshift/knative-v0.15.2:kn-cli-artifacts
```

Comment:
there is another reference of this image at line no:523 in `spec.relatedImages` section which should be updated as well